evm/gas-service.js

```diff
@@ -0,0 +1,414 @@
+'use strict';
+
+const { ethers } = require('hardhat');
+const {
+    getDefaultProvider,
+    utils: { formatEther, parseEther },
+    Contract,
+    BigNumber,
+} = ethers;
+const { Command, Option } = require('commander');
+const {
+    printInfo,
+    printWalletInfo,
+    printWarn,
+    mainProcessor,
+    prompt,
+    getGasOptions,
+    wasEventEmitted,
+    isValidAddress,
+    validateParameters,
+} = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const IAxelarGasService = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarGasService.json');
+const { getWallet } = require('./sign-utils');
+
+async function getGasInfo(env, chain, destinationChains) {
+    const apiUrl = env === 'testnet' ? 'https://testnet.api.axelarscan.io/gmp/getFees' : 'https://api.axelarscan.io/gmp/getFees';
+
+    return Promise.all(
+        destinationChains.map((destinationChain) => {
+            let feeType = 0;
+            let blobBaseFee = 0;
+
+            switch (destinationChain.toLowerCase()) {
+                case 'ethereum-sepolia':
```

Comment:
L1 name might not be ethereum on the network. Push ethereum's config under the `ethereum` key as well if it wasn't done

Reply:
I moved everything to config, no hardcoded chain names anymore.

Reply:
we still need to post L1 info under `ethereum` [key](https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/main/contracts/gas/InterchainGasEstimation.sol#L61) even if ethereum's axelarId is not `ethereum`